code/game/atoms_movable.dm

```diff
@@ -78,6 +78,11 @@
 	locking_categories      = null
 	locking_categories_name = null
 
+	if((flags & HEAR) && !ismob(src))
```

Comment:
This does mean atoms can't toggle the HEAR flag in their lifetimes without breaking this.